publish/src/commands/deploy.js

```diff
@@ -1191,632 +1223,970 @@ const deploy = async ({
-						const namesReadable = names.map(w3utils.hexToUtf8);
-
-						// track requirements to log out later
-						namesReadable.forEach(
-							name =>
-								(contractResolverRequirements[name] = [contract].concat(
-									contractResolverRequirements[name]
-								))
-						);
+	if (!useOvm) {
+		console.log(gray(`\n------ DEPLOY MULTI COLLATERAL ------\n`));
 
-						return namesReadable;
-					})
-				)
-		);
+		const managerState = await deployer.deployContract({
+			name: 'CollateralManagerState',
+			args: [account, account],
+		});
 
-		const allRequiredAddresses = Array.from(
-			// create set to remove dupes
-			new Set(
-				allRequiredAddressesInContracts
-					// flatten into one array
-					.reduce((memo, entry) => memo.concat(entry), [])
-					// and remove blanks
-					.filter(entry => entry)
-					// now filter out any externals or alternates with a colon
-					.filter(entry => {
-						if (/:/.test(entry)) {
-							skipResolverSync = skipResolverSync.concat(contractResolverRequirements[entry]);
-							console.log(
-								redBright(
-									`⚠⚠⚠ WARNING: Skipping AddressResolver requirement of "${entry}" (from ${contractResolverRequirements[entry]})`
-								)
-							);
-							return false;
-						}
-						return true;
-					})
-					// SystemSettings isn't required anywhere but necessary for us to be able to
-					// write to FlexibleStorage below via "setExchangeFeeRates()"
-					.concat(['SystemSettings'])
-			)
-		).sort();
-
-		// now map these into a list of names and addresses
-		const expectedAddressesInResolver = allRequiredAddresses.map(name => {
-			const contract = deployer.deployedContracts[name];
-			// quick sanity check of names in expected list
-			if (!contract) {
-				throw Error(
-					`Error setting up AddressResolver: cannot find one of the contracts listed as required in a contract: ${name} in the list of deployment targets`
-				);
+		collateralManager = await deployer.deployContract({
+			name: 'CollateralManager',
+			args: [
+				addressOf(managerState),
+				account,
+				addressOf(readProxyForResolver),
+				collateralManagerDefaults['MAX_DEBT'],
+				collateralManagerDefaults['BASE_BORROW_RATE'],
+				collateralManagerDefaults['BASE_SHORT_RATE'],
+			],
+		});
+
+		if (managerState && collateralManager) {
+			await runStep({
+				contract: 'CollateralManagerState',
+				target: managerState,
+				read: 'associatedContract',
+				expected: input => input === addressOf(collateralManager),
+				write: 'setAssociatedContract',
+				writeArg: addressOf(collateralManager),
+			});
+		}
+
+		const collateralStateEth = await deployer.deployContract({
+			name: 'CollateralStateEth',
+			source: 'CollateralState',
+			args: [account, account],
+		});
+
+		collateralEth = await deployer.deployContract({
+			name: 'CollateralEth',
+			args: [
+				addressOf(collateralStateEth),
+				account,
+				addressOf(collateralManager),
+				addressOf(readProxyForResolver),
+				toBytes32('sETH'),
```

Comment:
Nedd change to `zBNB`
```suggestion
				toBytes32('zBNB'),
```